doc/ref_kernel.rst

```diff
@@ -3,6 +3,69 @@
 Reference: Loopy's Model of a Kernel
 ====================================
 
+What Types of Computation can a Loopy Program Express?
+------------------------------------------------------
+
+Loopy programs consist of an a-priori unordered set of statements, operating
+on :math:`n`-dimensional array variables.
+
+Arrays consist of "plain old data" and structures thereof, as describable
+by a :class:`numpy.dtype`.  The n-dimensional shape of these arrays is
+given by a tuple of expressions at most affine in parameters that are
+fixed for the duration of program execution.
+Each array variable in the program is either an argument or a temporary
+variable.  A temporary variable is only live within the program, while
```

Comment:
Thanks for taking a look. This is fine IMO, as "live" is being used as an adjective, as in https://en.wikipedia.org/wiki/Live_variable_analysis.

As for the FAQ note, I think it would be out of place, since IMO
- the content of the FAQ note is implied by the text here. (let me know if you disagree), and
- the FAQ note is more informal advice, whereas this is meant to be technically precise.

Reply:
Oh I see. I read it as the verb 'live' (as in, 'be alive'). That all makes sense to me.